app/routes/_index/route.tsx

```diff
@@ -32,7 +32,6 @@ export async function loader({ params }: LoaderArgs) {
 
   for (let i = 0; i < 4; i++) {
     const post = await getPost(blogs[i].slug);
-
```

Comment:
¿No te funciona con un solo `for` todos los posts? El backend obtiene los post de forma descendente por fecha de creación.